.github/workflows/merlin.yml

```diff
@@ -400,8 +400,9 @@ jobs:
       K3D_CLUSTER: merlin-cluster
       LOCAL_REGISTRY_PORT: 12345
       LOCAL_REGISTRY: "dev.localhost"
+      DOCKER_REGISTRY: "dev.localhost:12345"
       INGRESS_HOST: "127.0.0.1.nip.io"
-      MERLIN_CHART_VERSION: 0.13.4
+      MERLIN_CHART_VERSION: 0.13.18
```

Comment:
The e2e tests were using a relatively older version of the Merlin helm chart and I've updated it to the latest version.